build.gradle

```diff
@@ -337,6 +337,8 @@ dependencies {
   implementation fileTree(dir: 'libs', include: ['*.jar'])
   implementation 'com.simprints:LibSimprints:1.0.11'
   implementation 'com.github.Mariovc:ImagePicker:1.2.2'
+  implementation 'com.github.dimagi:rd-toolkit:0.9.0'
```

Comment:
Hi @mrsarm I tried moving the class to inside the webview folder but still getting the same error, I wonder if because the lib is introduced here and taken into account in all builds 🤔
<img width="1312" alt="Screen Shot 2021-05-13 at 3 57 38 pm" src="https://user-images.githubusercontent.com/66472237/118103403-13345e00-b404-11eb-922a-4b47d400cb58.png">

BTW I tried building for Android 9 but geting an error where sdk isn't reaching the minimum supported by our webview build:
<img width="636" alt="Screen Shot 2021-05-13 at 3 57 10 pm" src="https://user-images.githubusercontent.com/66472237/118103627-5393dc00-b404-11eb-806e-7668917bbd85.png">

I think maybe we should make a flavour for this partner until we make webview fully supported across android versions, what do you think?

Reply:
To make it compile you have to change the `implementation` call with `webviewImplementation` with the libraries added:

```diff
-   implementation 'com.github.dimagi:rd-toolkit:0.9.0'
-   implementation 'androidx.core:core-ktx:1.0.1'
+   webviewImplementation 'com.github.dimagi:rd-toolkit:0.9.0'
+   webviewImplementation 'androidx.core:core-ktx:1.0.1'
```

So they are only included in the Webview flavors, and remove the `android.enableJetifier=true` property added that is not compatible with XWalk. If that property is really needed, you should find a way to only add it when compiling Webview flavors.

Reply:
:thinking: removing `android.enableJetifier=true` causes the project to compile but not to build in Webview flavors, so you have to figure out how to add that property only for Webview flavors, so far I couldn't find a simple way to do it, it is easy from the command line via arguments but must be a way to override the property regarding the flavor.

Reply:
Thanks Mariano, that's good stuff!! 
Yes, we need the `android.enableJetifier=true` because the RDToolkit lib is using the new [android support lib](https://developer.android.com/jetpack/androidx) (backward compatible) and if we use the old android.support.v4 lib (not maintenance since 2018) throws an error of incompatibility.
Actually, since we're using both flags, then maybe I dont need to import the androidx manually (trying out...)

Reply:
I tried to add this flag inside the webview flavor (build.gradle) but it's not working, the posts I've found say that these android flags must go in the gradle.properties 
At the end, I moved it to `deploy-flavour` command in makefile and the build works, but then the `make test` is failing. 
If I add it in the `make test` then it works for webview but not for xwalk. I'm trying to find where we are configuring the test.

Reply:
This is fixed, I've put back the `android.enableJetifier=true` in gradle.properties to migrate third parties with the new android [support lib](https://developer.android.com/jetpack/androidx) in webview since that's the way in the future, and disable the flag in xwalk